doc/source/dsintro.rst

```diff
@@ -856,8 +856,7 @@ DataFrame objects with mixed-type columns, all of the data will get upcasted to
 From DataFrame using ``to_panel`` method
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This method was introduced in v0.7 to replace ``LongPanel.to_long``, and converts
-a DataFrame with a two-level index to a Panel.
+``to_panel`` converts a DataFrame with a two-level index to a Panel.
```

Comment:
can you add a referencde to the section where panel is deprecated.

Reply:
There is a deprecation warning a bit above, so it's too much adding it here also IMO. I changed a note that calls on people to contribute to panels, though, as that isnt relevant anymore.